plugin/pkg/auth/authenticator/token/tokenfile/tokenfile_test.go

```diff
@@ -29,8 +29,10 @@ func TestTokenFile(t *testing.T) {
 	auth, err := newWithContents(t, `
 token1,user1,uid1
 token2,user2,uid2
-token3,user3,uid3,group1,group2
-token4,user4,uid4,group2
+token3,user3,uid3,"group1,group2"
+token4,user4,uid4,"group2"
+token5,user5,uid5,group5
+token6,user6,uid6,group5,group6
```

Comment:
call the last column "otherdata" or something... "group6" not getting included in the user's groups looks like a bug